test/Constellation/Node/Storage/BerkeleyDb/Test.hs

```diff
@@ -1,8 +1,23 @@
 {-# LANGUAGE NoImplicitPrelude #-}
+{-# LANGUAGE Strict #-}
```

Comment:
Switched over to StrictData in other files at least until the difference can be benchmarked (Strict may be too aggressive, force evaluation of log arguments even if log statements are muted, etc.)